src/app/admin/pages/learning-objects/learning-objects.component.ts

```diff
@@ -1,69 +1,71 @@
 import { Component, OnInit } from '@angular/core';
+import { LearningObjectService as publicService } from 'app/cube/learning-object.service';
```

Comment:
Overall the fact that we are needing to alias like this smells of bad design, however I think a refactoring here might be out of scope for this PR. I do however think that the alias names should:
1. Be descriptive of the service type, not just generically a "service"
2. Maintain the casing of the aliased member, instead of converting to camelCase